File: java/program72.java
import java.util.*;

class program72
{
    public static boolean CheckChar(char ch)
    {
        if((ch >= 'A') && (ch <= 'Z') || (ch >= 'a') && (ch <= 'z'))
        {
            return true;
        }
        else
        {
            return false;
        }
    }

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        boolean bRet = false;

        System.out.println("Enter character");
        char ch = sobj.next().charAt(0);

        bRet = CheckChar(ch);
        if(bRet == true)
        {
            System.out.println(ch+" is character");
        }
        else
        {
            System.out.println(ch+" is not character");
        }
    }
}

File: cpp/program72.cpp
#include<iostream>

bool CheckCapital(char ch)
{
    if((ch >= 'A') && (ch <= 'Z'))
    {
        return true;
    }
    else 
    {
        return false;
    }
}

int main()
{
    char ch = '\0';
    bool bRet = false;

    std::cout<<"Enter character \n";
    std::cin>>ch;

    bRet = CheckCapital(ch);
    if(bRet == true)
    {
        std::cout<<ch<<" is capital letter \n";
    }
    else
    {
        std::cout<<ch<<" is NOT capital letter \n";
    }

    return 0;
}